spec/unit/shoulda/matchers/action_controller/route_matcher_spec.rb

```diff
@@ -64,6 +78,14 @@
             to("#{controller_path}#show", id: 1)
         end
       end
+
+      context 'when route has the format' do
+        it 'accepts' do
+          expect(controller_with_defined_routes).
+            to route(:post, "/#{controller_path}").
+            to("#{controller_path}#create", format: 'json')
+        end
+      end
```

Comment:
Same thing here, seems we need to test that it rejects if format is not specified.